yjit/src/codegen.rs

```diff
@@ -6554,6 +6554,41 @@ fn gen_setclassvariable(
     KeepCompiling
 }
 
+fn gen_getconstant(
+    jit: &mut JITState,
+    ctx: &mut Context,
+    asm: &mut Assembler,
+    _ocb: &mut OutlinedCb,
+) -> CodegenStatus {
+
+    let id: *const ID = jit_get_arg(jit, 0).as_ptr();
```

Comment:
🤔 I think this works, but this is not a pointer, so things like `as_usize()` might be more appropriate (like what `gen_getglobal` does)